### ISSUE TITLE
java.lang.NoSuchMethodError: No virtual method load(Ljava/io/File;)

### ISSUE THREAD

**Issue by Paridel** [nileshpambhar/MultiPhotoPicker#3]
error while run this library

java.lang.NoSuchMethodError: No virtual method load(Ljava/io/File;)Lcom/bumptech/glide/DrawableTypeRequest; in class Lcom/bumptech/glide/RequestManager; or its super classes (declaration of 'com.bumptech.glide.RequestManager' appears in /data/app/cg.polytechbrazza.ndako-2/split_lib_dependencies_apk.apk)

**Comment by Paridel** [comment]
I use Glide 'com.github.bumptech.glide:glide:4.3.1'

**Comment by ghost** [comment]
Yeah i'm having that error too. His glide version is 3.8.1 please update your glide library. I cant afford to downgrade anymore. Thanks!
